### ISSUE TITLE
time4tv channel 

### ISSUE THREAD

**Issue by owencz1998** [hexated/cloudstream-extensions-hexated#78]
Is there away to add sky sport box office using this link mate 

https://iframe.tv/lala/index.php?stream=125952231?sid=1592438

**Comment by owencz1998** [comment]
And sorry for the adds haha

**Comment by hexated** [comment]
> Is there away to add sky sport box office using this link mate 
> 
> https://iframe.tv/lala/index.php?stream=125952231?sid=1592438

I don't think so. I can hardcoded but it hard to follow if website changed

**Comment by owencz1998** [comment]
No worries dude I think it's hard finding a site that has sky box office uk if i could find a iptv then i would just do that 🤣
